tests/python/unittest/test_target_custom_datatypes.py

```diff
@@ -128,7 +128,7 @@ def test_bfloat_add_and_cast_FloatImm():
     Z = topi.cast(
         topi.add(
             topi.cast(X, dtype="custom[bfloat]16"),
-            tvm.tir.FloatImm("custom[bfloat]16", 1.5)),
+            tvm.tir.FloatImm("float", 1.5)),
```

Comment:
There is a reason why we did not add type check to the codebas here(to support custom data types) as show in here.

Reply:
I am sorry, i did not understand completely the logic behind it. I mean how FloatImm can support datatype other than float. It looks like some potential hole. That is why i added check. Anyway if it is on purpose, then i will ignore it.